proto/bff/models/v1/models.proto

```diff
@@ -20,7 +20,8 @@ message Organization {
     DirectoryRecord testnet = 10;
     DirectoryRecord mainnet = 11;
 
-    reserved 12; // Reserved for collaborators
+    // Collaborators
+    map<string, Collaborator> collaborators = 20;
```

Comment:
Did you not want to use the reserved field 12?

Reply:
Oops, typo on my part.